src/routes/institution-route.js

```diff
@@ -0,0 +1,13 @@
+import express from 'express'
+import * as controller from '../controllers/institution-controller.js'
+
+const institutionRouter = express.Router({ mergeParams: true })
+
+institutionRouter.get('/', controller.fetchProjectInstitutions)
+institutionRouter.get('/search', controller.searchInstitutions)
+
+institutionRouter.post('/create', controller.addInstitutionToProject)
+institutionRouter.post('/createInstitution', controller.createInstitution)
```

Comment:
Rather than `create` and `createInstitution`, you can rename it to `add` and `create` which is more intutitive.